docs/extend/modular-resources/examples/odrive.md

```diff
@@ -0,0 +1,281 @@
+---
+title: "Add an ODrive motor as a Modular Resource"
+linkTitle: "ODrive"
+weight: 40
+type: "docs"
+description: "How to add an ODrive motor with serial or canbus as a modular resource of your robot."
+tags: ["motor", "odrive", "canbus", "serial", "module", "modular resources", "Python", "python SDK"]
+# SMEs: Kim, Martha, Rand
+---
+
+The [Viam GitHub](https://github.com/viamrobotics/odrive) provides an implementation of ODrive Robotics' [ODrive S1](https://odriverobotics.com/shop/odrive-s1) motor driver as a modular resource [extending](/extend/modular-resources/) the [motor API](/components/motor#API).
+
+[Prepare](#prepare-your-odrive) your ODrive and [download](#requirements) and [configure](#configuration) the module to add an `odrive-serial` or `odrive-canbus` [motor](/components/motor/) {{< glossary_tooltip term_id="resource" text="resource" >}} to your robot.
+
+{{% alert title="Note" color="note" %}}
+
+This module is only implemented for use with the [Viam Python SDK](https://python.viam.dev/).
+The methods other SDKs provide for the [motor API](/components/motor#API) will not work with this module.
+
+{{% /alert %}}
+
+### Prepare your ODrive
+
+1. Read through the [ODrive documentation](https://docs.odriverobotics.com/v/latest/getting-started.html) to wire, calibrate, and configure your ODrive natively.
+
+{{% alert title="Tip" color="tip" %}}
+
+This configuration remains on the same ODrive motor controller across reboots, and only changes when you go through the configuration of the ODrive again.
+
+To set the native configuration dynamically, find and copy the path to the motor's `config.json` file to provide in configuration as the attribute `"odrive_config_file"`.
```

Comment:
I believe there's instructions on how to do this in the README, but I don't actually think you can "find" the config file. I'm pretty sure you have to run `odrivetool backup-config /path/to/where/config.json`

Reply:
also, we should probably emphasize that adding the file in the config as `"odrive_config_file"` is completely optional (and honestly not ideal for canbus implementations because then there still has to be a serial connection to use the config file as expected)

Reply:
Ok sounds good to me